recipes/hdijupyterutils/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "hdijupyterutils" %}
+{% set version = "0.12.1" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "26cdd497aaf2cfe5fad807090ddade6da90d761ef1e8352b5c5890ad9d38162c" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    {{ hash_type }}: {{ hash_val }}
+
+build:
+    number: 0
+    script: python setup.py install
```

Comment:
Since `setuptools` is available, this [should probably use](https://conda-forge.org/docs/meta.html#single-version-externally-managed) `python setup.py install --single-version-externally-managed --record record.txt`.